entry.go

```diff
@@ -617,14 +618,14 @@ func StartReverse(config *EntryConfiguration, wallet *nkn.Wallet) error {
 				continue
 			}
 			if bytes.Equal(buffer[:PrefixLen], []byte{PrefixLen - 1: 0}) && n > PrefixLen {
-				if encrypted {
-					continue
-				}
 				connMetadata, err := parseUDPConnMetadata(buffer[PrefixLen:n])
```

Comment:
Yes, the ping msg is encrypted since it will be sent after the UDP conn encrypted

> metadata will be parsed even if `encrypted == true`, is that expected?